### PR TITLE
Add perf-dash prefixes to prow config.

### PR DESCRIPTION
**Pull request by porridge**
This is a step towards solving https://github.com/kubernetes/contrib/issues/2812
@shyamjvs FYI
/cc @BenTheElder @krzyzacy

### CODE REVIEW

**Comment by porridge** [comment]
Fixed indentation.

File: prow/config.yaml [hunk 1]
```diff
@@ -11514,6 +11514,8 @@ periodics:
 - cron: '1 13 * * 2,4,6' # Run at 5:01PST (13:01 UTC) on even days
   agent: kubernetes
   name: ci-kubernetes-e2e-gce-large-performance
+  tags:
+  - "perfDashPrefix: gce-2kNodes-master"
```
Comment (krzyzacy) [inline prow/config.yaml]: I don't think you need quotes?

**Comment by porridge** [comment]
I do. See discussion on the bug.

**Pull request merged**